lib/YaST/Bootloader/BootloaderSettingsController.pm

```diff
@@ -79,6 +81,14 @@ sub write_to_partition {
     $self->get_boot_code_options_page->press_next();
 }
 
+sub disable_grub_timeout_navigating_tabs {
+    my ($self) = @_;
+    $self->get_boot_code_options_page->is_shown();
```

Comment:
I guess just `$self->get_boot_code_options_page()` you don't need to run `is_shown()` twice.